gravitee-node-notifier/src/main/java/io/gravitee/node/notifier/trigger/NotificationTrigger.java

```diff
@@ -87,16 +89,21 @@ public NotificationTrigger(
     }
 
     public void start() {
+        started = true;
         scheduleNextAttempt();
     }
 
     private void scheduleNextAttempt() {
-        this.scheduledTaskId = this.vertx.setTimer(computeNextAttempt(), this);
+        if (started) {
+            this.scheduledTaskId = this.vertx.setTimer(computeNextAttempt(), this);
+        }
     }
 
     public void stop() {
-        if (this.scheduledTaskId != null && this.vertx.cancelTimer(this.scheduledTaskId)) {
+        if (started && this.scheduledTaskId != null && this.vertx.cancelTimer(this.scheduledTaskId)) {
```

Comment:
It probably requires synchronizing the started boolean so we ensure there is not a thread setting a timer while another is moving the `started` boolean to `false` (which will end in the same kind of situation)

Reply:
Atomicboolean would be cheaper don’t u think?

Reply:
Totally agree 👍